src/test/java/com/exasol/errorcodecrawlermavenplugin/ExasolErrorCrawlerTest.java

```diff
@@ -0,0 +1,44 @@
+package com.exasol.errorcodecrawlermavenplugin;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.containsInAnyOrder;
+import static org.hamcrest.Matchers.equalTo;
+import static org.junit.jupiter.api.Assertions.assertAll;
+
+import java.nio.file.Path;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.junit.jupiter.api.Test;
+
+import com.exasol.errorcodecrawlermavenplugin.model.ExasolError;
+
+class ExasolErrorCrawlerTest {
+    private static final Path PROJECT_DIRECTORY = Path.of(".").toAbsolutePath();
+    private static final ErrorCrawler ERROR_CRAWLER = new ErrorCrawler(PROJECT_DIRECTORY, new String[] {});
+
+    @Test
+    void test() {
+        final ErrorCrawler.Result result = ERROR_CRAWLER.crawl(
+                Path.of("src/test/java/com/exasol/errorcodecrawlermavenplugin/examples/Test1.java").toAbsolutePath());
```

Comment:
I usually prefer to have the data that is used in the unit tests defined here. I find it a bit cumbersome the need of going to check another file to see the data input, in this case Test1.

Would it make sense to define this data here somehow?

Reply:
I don't see a possibility to move it here...